phoenix-core/src/test/java/org/apache/phoenix/index/IndexUpgradeToolTest.java

```diff
@@ -27,34 +27,57 @@
 import org.apache.commons.cli.CommandLine;
 import org.apache.phoenix.mapreduce.index.IndexUpgradeTool;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
+import org.mockito.Mock;
+import org.mockito.Mockito;
 
 @RunWith(Parameterized.class)
 public class IndexUpgradeToolTest {
     private static final String INPUT_LIST = "TEST.MOCK1,TEST1.MOCK2,TEST.MOCK3";
     private final boolean upgrade;
+    private static final String VERIFY_VALUE = "AFTER";
+    private IndexUpgradeTool indexUpgradeTool=null;
+    private String outputFile;
 
     public IndexUpgradeToolTest(boolean upgrade) {
         this.upgrade = upgrade;
     }
 
+    @Before
+    public void setup() {
+        outputFile = "/tmp/index_upgrade_" + UUID.randomUUID().toString();
+        String [] args = {"-o", upgrade ? UPGRADE_OP : ROLLBACK_OP, "-tb",
+                INPUT_LIST, "-lf", outputFile, "-d", "-v", VERIFY_VALUE};
+        indexUpgradeTool = new IndexUpgradeTool();
+        CommandLine cmd = indexUpgradeTool.parseOptions(args);
+        indexUpgradeTool.initializeTool(cmd);
+    }
+
     @Test
     public void testCommandLineParsing() {
+        Assert.assertEquals(indexUpgradeTool.getDryRun(),true);
+        Assert.assertEquals(indexUpgradeTool.getInputTables(), INPUT_LIST);
+        Assert.assertEquals(indexUpgradeTool.getOperation(), upgrade ? UPGRADE_OP : ROLLBACK_OP);
+        Assert.assertEquals(indexUpgradeTool.getLogFile(), outputFile);
+    }
 
-        String outputFile = "/tmp/index_upgrade_" + UUID.randomUUID().toString();
-        String [] args = {"-o", upgrade ? UPGRADE_OP : ROLLBACK_OP, "-tb",
-                INPUT_LIST, "-lf", outputFile, "-d"};
-        IndexUpgradeTool iut = new IndexUpgradeTool();
-
-        CommandLine cmd = iut.parseOptions(args);
-        iut.initializeTool(cmd);
-        Assert.assertEquals(iut.getDryRun(),true);
-        Assert.assertEquals(iut.getInputTables(), INPUT_LIST);
-        Assert.assertEquals(iut.getOperation(), upgrade ? UPGRADE_OP : ROLLBACK_OP);
-        Assert.assertEquals(iut.getLogFile(), outputFile);
+    @Test
+    public void testIfVerifyOptionIsPassedToTool() {
+        if (!upgrade) {
+            return;
+        }
+        Assert.assertEquals("", VERIFY_VALUE, indexUpgradeTool.getVerify());
+        String [] values = indexUpgradeTool.getIndexToolArgValues(Mockito.anyString(),
```

Comment:
It's unusual to use Mockito.anyString() in a call to something that isn't setting up a mock. Any reason these can't just be dummy values?

Reply:
I can create a dummy value for the entire test and use it wherever necessary. Mock.anyString() was the way I thought to use it as a dummy. :)